betaflight/firmware-presets
# New RC_LINK presets for Ghost and TBS

Experimental presets based on generic values until we get better values from BlackBox recordings.

if this PR is not merged yet then:
add custom source in the configurator:
https://github.com/limonspb/firmware-presets
branch name: ghost_test

If PR is merged already then proceed with the following (using official preset source):

For testers, who are willing to provide BlackBox to improve RC_LINK presets for Ghost/Tracer/Crossfire:
1. Select RC mode you want to test in Ghost/Crossfire or Tracer
2. Select the existing RC_LINK preset for the mode you want to test.
3. Select Race or Freestyle option.
4. Set logging in the following way:
![image](https://user-images.githubusercontent.com/2925027/157808957-5df1bb4e-3900-442e-8782-8211baecabbe.png)
5. **Take props OFF** or don't even power the motors (make sure RX is powered)
6. "Arm" your drone.
7. start slow, do like 0.5hz roll moves. Stay inside one-half of the range and avoid hitting center or limits. Then slowly get faster and faster.
8. Then some ‘imaginary’ flips or rolls.  and some ‘pretend flying

Download blackbox and share it here.
Mention what mode you were testing and which preset did you apply.
